### DISCUSSION TITLE
Clarification on Output Compatibility with Non-OpenAI LLMs

### DISCUSSION THREAD

**Discussion by codebrain001** [crewAIInc/crewAI#1095] [Q&A]
Hello,

Could you clarify for me the compatibility of the output formats mentioned in the documentation? Specifically, I have a question regarding the following options:

1. Output Pydantic (optional)

- Field: output_pydantic
- Description: Outputs a Pydantic model object, requiring an OpenAI client. Only one output format can be set.

2. Output JSON (optional)

- Field: output_json
- Description: Outputs a JSON object, requiring an OpenAI client. Only one output format can be set.
My question is:

Does the requirement for an OpenAI client for these output formats mean that they cannot be used with other LLMs, such as Claude models from Anthropic?

Understanding the constraints and compatibility is crucial for integrating these models into our existing workflows that utilize different LLMs. Any insights or additional documentation references would be highly appreciated.

Thank you!

**Comment by andresperezrobinson** [comment]
Would like to expand on this question, as I'm having issues with output_json while using Llama3.
